lib/commands/index.js

```diff
@@ -7,6 +7,7 @@ import screenShotCmds from './screenshot';
 import elementCmds from './element';
 import gestureCmds from './gesture';
 import navigationCmds from './navigation';
+import executeCmds from './execute';
```

Comment:
Vague

Reply:
standard name they are using in other drivers